src/.vuepress/sidebar/en.ts

```diff
@@ -58,6 +58,10 @@ export const enSidebar = sidebar({
           prefix: "telemetry/",
           link: "telemetry/",
           children: [
+            {
+              text: "Logs",
+              link: "logs.md",
+            },
```

Comment:
IMHO, these are (structured) `events` not `logs`, but we are _logging_ these events via `exporters`.